ansible_collections/arista/avd/roles/eos_designs/python_modules/ip_addressing/avdipaddressing.py

```diff
@@ -249,5 +264,9 @@ def vtep_ip(self) -> str:
                 loopback_ipv4_offset=self._loopback_ipv4_offset,
             )
 
-        offset = self._id + self._loopback_ipv4_offset
+        if self._is_host_subnet(self._vtep_loopback_ipv4_pool):
```

Comment:
Since this is behavioral change, we should update the description on the pools covered by this change.
You could just have entered `-1` as the offset value.

Reply:
not sure what you mean about `-1` it would not work right if id is 10 ? the value has to be 0 for it to work as you pointed out